mathesar_ui/src/systems/databases/create-database/ConnectExistingDatabase.svelte

```diff
@@ -0,0 +1,108 @@
+<script lang="ts">
+  import { _ } from 'svelte-i18n';
+
+  import type { Database } from '@mathesar/api/rpc/databases';
+  import {
+    FieldLayout,
+    FormSubmit,
+    makeForm,
+    requiredField,
+  } from '@mathesar/components/form';
+  import Field from '@mathesar/components/form/Field.svelte';
+  import { databasesStore } from '@mathesar/stores/databases';
+  import {
+    NumberInput,
+    PasswordInput,
+    portalToWindowFooter,
+  } from '@mathesar-component-library';
+
+  import {
+    type InstallationSchema,
+    getSampleSchemasFromInstallationSchemas,
+  } from './createDatabaseUtils';
+  import InstallationSchemaSelector from './InstallationSchemaSelector.svelte';
+
+  export let onCancel: () => void;
+  export let onSuccess: (db: Database) => void;
+
+  const databaseName = requiredField('');
+  const host = requiredField('localhost');
+  const port = requiredField(5432);
+  const role = requiredField('');
+  const password = requiredField('');
+  const installationSchemas = requiredField<InstallationSchema[]>(['internal']);
+  const form = makeForm({ databaseName, installationSchemas });
```

Comment:
I think `makeForm` should include all the form fields. That way the "Create Database" button doesn't get enabled until all the fields are filled out. Was there a reason you left some of the fields out of the form?

Reply:
Good catch. I missed them while adding the fields. https://github.com/mathesar-foundation/mathesar/pull/3711/commits/e6a70d922c0f6527acf96bb96f60d646824dbc63 fixes this.